src/dmd/denum.d

```diff
@@ -471,5 +470,3 @@ bool isSpecialEnumIdent(const Identifier ident) @nogc nothrow
             ident == Id.__c_long_double ||
             ident == Id.__c_wchar_t;
 }
-
-
```

Comment:
for a smaller diff, revert your changes in this module

Reply:
Done.